docs/source/api_reference/regression.rst

```diff
@@ -5,9 +5,14 @@ Time series regression
 
 The :mod:`sktime.regression` module contains algorithms and composition tools for time series regression.
 
-.. automodule:: sktime.regression
-    :no-members:
-    :no-inherited-members:
+All current sktime Regressors can be listed as follows
+
+```
+from sktime.registry import all_estimators
+cls = all_estimators(estimator_types="regressor")
+names = [i for i, _ in cls]
+print(names)
+```
```

Comment:
This does not seem to be rendering as intended, it's all on one line. i.e. `from sktime.registry import all_estimators cls = all_estimators(estimator_types="regressor") names = [i for i, _ in cls] print(names)`.

Reply:
how is it now? cant really tell in the preview